subprojects/core/src/main/java/org/gradle/api/internal/tasks/options/AbstractOptionElement.java

```diff
@@ -32,7 +33,7 @@
     private final Class<?> optionType;
     private final NotationParser<CharSequence, ?> notationParser;
 
-    public AbstractOptionElement(String optionName, Option option, Class<?> optionType, Class<?> declaringClass, NotationParser<CharSequence, ?> notationParser) {
+    AbstractOptionElement(String optionName, Option option, Class<?> optionType, Class<?> declaringClass, NotationParser<CharSequence, ?> notationParser) {
```

Comment:
Thanks for the info. We usually don't do that. Let's revert those changes for now. It will also make it easier to review the changes and get them into the ready-state.